content/rc/security/database-security/tls-ssl.md

```diff
@@ -7,17 +7,15 @@ categories: ["RC"]
 aliases: /rc/administration/security/securing-redis-cloud-connections
 ---
 
-Transport Layer Security (TLS), commonly called "SSL", ensures the privacy of data sent between applications and their Redis databases. TLS is available in Redis Cloud Pro and Redis Cloud Ultimate.
-
-## Using TLS with Redis Cloud
+Transport Layer Security (TLS), commonly called "SSL", ensures the privacy of data sent between applications and their Redis databases. TLS is available in Redis Cloud Flexible and Annual subscriptions.
+\
```

Comment:
@lanceleonard Is this "\" intentional?

Reply:
Not by me.  :-)

(I hadn't done a copy edit on the content.  Was trying to only update the plan names.  The more recent commits include a quick copy edit and updated images for the certificate authorization buttons.)